lib/datadog/core/remote/transport/http/config.rb

```diff
@@ -130,8 +132,25 @@ def initialize(key)
                 end
               end
 
+              # Base class for Remote Configuration Config errors
+              class ConfigError < StandardError
+              end
+
+              # When the agent returned an error response to our request
+              class AgentErrorResponse < ConfigError
+                def initialize(code, body)
+                  truncated_body = if body.length > 1000
+                                     "#{body[0...800]}...#{body[800...1000]}"
+                                   else
+                                     body
+                                   end
```

Comment:
Just to be more consistent with magic numbers, does it make sense to have as constants?